sdcm/cluster.py

```diff
@@ -2608,6 +2610,9 @@ def check_node_health(self, retries: int = CHECK_NODE_HEALTH_RETRIES) -> None:
                     event.publish()
                 break
 
+            for event in events:
+                event.dont_publish()
```

Comment:
it should be:

```python
event.dont_publish()
```

The whole idea of having generators here is to stop making health checks on the first error. This change breaks this thing.

Reply:
True